notifications_utils/version.py

```diff
@@ -5,4 +5,4 @@
 # - `make version-minor` for new features
 # - `make version-patch` for bug fixes
 
-__version__ = '55.0.0'  # 5568605a0a138ed2231ae2ae2a13beae
+__version__ = '55.1.0'  # aaa1b86c488c4e25c842f0a443f9a26b
```

Comment:
I would have expected this to be a major upgrade since it's a breaking change to admin and api. Shall we make this version 6.0.0?

Reply:
There shouldn't be any breaking changes in here:

- The decorator is new and will only be used in Admin.
- The redis module changes should be backwards compatible.
- The rest of the changes are just to the tests.

The module change is a big one and not something I've had much practice at. Is there something I missed there?